.github/workflows/ci.yml

```diff
@@ -0,0 +1,119 @@
+name: CI
+on: [push]
+jobs:
+  python-tests:
+    runs-on: ubuntu-latest
+
+    services:
+      # Label used to access the service container
+      db:
+        # Docker Hub image
+        image: postgres:11.6
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+
+      redis:
+        image: redis:5.0.8
+        ports:
+          - 6379:6379
+
+      elastic:
+        image: docker.elastic.co/elasticsearch/elasticsearch:6.5.4
+        env:
+          network.host: "0.0.0.0"
+          http.cors.enabled: "true"
+          http.cors.allow-origin: "*"
+          rest.action.multi.allow_explicit_index: "false"
+          ES_JAVA_OPTS: -Xms512m -Xmx512m"
+        ports:
+          - 9200:9200
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Apt install
+        run: cat Aptfile | sudo xargs apt-get install
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+
+      - name: Install dependencies
+        run: pip install -r requirements.txt -r test_requirements.txt
+
+      - name: Code formatting
+        run: black --check .
+
+      - name: Lint
+        run: pylint ./**/*.py
+
+      - name: Tests
+        run: |
+          export MEDIA_ROOT="$(mktemp -d)"
+          tox
```

Comment:
am I right in reading this that the tests are now being run outside of docker? just want to make sure I'm grokking this correctly

Reply:
Yes, I was hoping it would run a bit faster not having to build the containers. Seeing a 2-3 minute speedup but i'm not sure if that's caused by this or different hardware.

Reply:
it's probably related to ditching the containers - I saw a similar speedup on travis from moving from docker to "bare metal" (ha) for the JavaScript tests